go-apps/meep-rnis/server/rnis.go

```diff
@@ -1850,66 +1934,18 @@ func createSubscriptionLinkList(subType string) *SubscriptionLinkList {
 	return subscriptionLinkList
 }
 
-func subscriptionLinkListSubscriptionsGET(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
-
-	var response InlineResponse2003
-
-	subscriptionLinkList := createSubscriptionLinkList("")
-
-	response.SubscriptionLinkList = subscriptionLinkList
-	jsonResponse, err := json.Marshal(response)
-	if err != nil {
-		log.Error(err.Error())
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	fmt.Fprintf(w, string(jsonResponse))
-}
-
-func subscriptionLinkListSubscriptionsCcGET(w http.ResponseWriter, r *http.Request) {
+func subscriptionLinkListSubscriptionsGet(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 
-	var response InlineResponse2003
-
-	subscriptionLinkList := createSubscriptionLinkList(cellChangeSubscriptionType)
-
-	response.SubscriptionLinkList = subscriptionLinkList
-	jsonResponse, err := json.Marshal(response)
-	if err != nil {
-		log.Error(err.Error())
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	fmt.Fprintf(w, string(jsonResponse))
-}
-
-func subscriptionLinkListSubscriptionsReGET(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
-
-	var response InlineResponse2003
-
-	subscriptionLinkList := createSubscriptionLinkList(rabEstSubscriptionType)
-
-	response.SubscriptionLinkList = subscriptionLinkList
-	jsonResponse, err := json.Marshal(response)
-	if err != nil {
-		log.Error(err.Error())
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-	w.WriteHeader(http.StatusOK)
-	fmt.Fprintf(w, string(jsonResponse))
-}
-
-func subscriptionLinkListSubscriptionsRrGET(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	u, _ := url.Parse(r.URL.String())
+	log.Info("url: ", u.RequestURI())
+	q := u.Query()
+	subType := q.Get("subscription_type")
 
-	var response InlineResponse2003
+	var response InlineSubscriptionLinkList
 
-	subscriptionLinkList := createSubscriptionLinkList(rabRelSubscriptionType)
+	log.Info("SIMON type ", subType)
```

Comment:
Remove logs like this.

Reply:
i realised too late.. merged 2 SIMON comments.. were removed